wapitiCore/net/page.py

```diff
@@ -702,8 +702,14 @@ def html_redirections(self):
             if meta_tag and meta_tag["http-equiv"].lower() == "refresh":
                 content_str = meta_tag["content"].lower()
                 url_eq_idx = content_str.find("url=")
+                content_str_length = len(meta_tag["content"])
+
                 if url_eq_idx >= 0:
-                    url = meta_tag["content"][url_eq_idx + 4:]
+                    if meta_tag["content"][url_eq_idx + 4] in ("\"", "'"):
+                        url_eq_idx += 1
+                        if meta_tag["content"][len(meta_tag["content"]) - 1] in ("\"", "'"):
```

Comment:
Can you please use `startswith` / `endswith` here, it will be more readable ? My code was a bit clumsy, feel free to refactor it

Reply:
Done :rocket: